.github/workflows/e2e_tests.yml

```
@@ -59,75 +59,105 @@ jobs:
-      - name: Install system dependencies for WebKit
-        # Some WebKit dependencies seem to lay outside the cache and will need to be installed separately
-        if: ${{ steps.check.outputs.should_run == 'true' && steps.playwright-cache.outputs.cache-hit == 'true' }}
-        run: pnpm exec playwright install-deps webkit
-
-      # This workflow is triggered for all deployments (liam-app, liam-erd-sample, liam-docs),
-      # but E2E tests are executed only for liam-app deployments now.
-      - name: Run e2e tests
-        if: steps.check.outputs.should_run == 'true'
-        run: pnpm exec playwright test --project="${{ matrix.browser }}"
+      - name: Slack Notification on Failure
+        if: ${{ should_run == 'true' && github.event.deployment.environment == 'Preview – liam-app' && failure() }}
```

actionlint (1.7.7)

undefined variable "should_run". available variables are "env", "github", "inputs", "job", "matrix", "needs", "runner", "secrets", "steps", "strategy", "vars"

(expression)